src/backend/distributed/planner/multi_explain.c

```diff
@@ -124,85 +118,54 @@ MultiExplainOneQuery(Query *query, IntoClause *into, ExplainState *es,
 	}
 #endif
 
-	/* handle local queries in the same way as ExplainOneQuery */
-	if (localQuery)
-	{
-		PlannedStmt *plan = NULL;
-
-		INSTR_TIME_SET_CURRENT(planStart);
+	/* plan query, just like ExplainOneQuery does */
+	INSTR_TIME_SET_CURRENT(planStart);
 
-		/* plan the query */
-		plan = pg_plan_query(query, cursorOptions, params);
+	/* plan the query */
+	plan = pg_plan_query(query, cursorOptions, params);
 
-		INSTR_TIME_SET_CURRENT(planDuration);
-		INSTR_TIME_SUBTRACT(planDuration, planStart);
+	INSTR_TIME_SET_CURRENT(planDuration);
+	INSTR_TIME_SUBTRACT(planDuration, planStart);
 
+	/* if not a distributed query, use plain explain infrastructure */
+	if (!HasCitusToplevelNode(plan))
+	{
 		/* run it (if needed) and produce output */
 		ExplainOnePlan(plan, into, es, queryString, params, &planDuration);
-
-		return;
-	}
-
-	/*
-	 * standard_planner scribbles on it's input, but for deparsing we need the
-	 * unmodified form. So copy once we're sure it's a distributed query.
-	 */
-	originalQuery = copyObject(query);
```

Comment:
What makes this copy no longer necessary? The elimination of the non-standard `EXPLAIN` modes? Is that comment just out of date?